wd_sched.c

```diff
@@ -501,8 +496,7 @@ struct wd_sched *wd_sched_rr_alloc(__u8 sched_type, __u8 type_num,
 		return NULL;
 
 	if (sched_type >= SCHED_POLICY_BUTT || !type_num) {
-		WD_ERR("invalid: sched_type is %u or type_num is %u!\n",
-		       sched_type, type_num);
+		WD_ERR("invalid: sched_type is %u or type_num is %u!\n", sched_type, type_num);
 		return NULL;
 	}
 
```

Comment:
Now that the scheduler has been modified, add this missing assignment as well:
sched->sched_policy = sched_type;
sched->name = sched_table[sched_type].name;

Reply:
Got it and I'll update this next version.
Thx!